tgui/packages/tgui/interfaces/CargoConsole.js

```diff
@@ -0,0 +1,241 @@
+import { flow } from 'common/fp';
+import { Fragment } from 'inferno';
+import { filter, sortBy } from 'common/collections';
+import { useBackend, useSharedState, useLocalState } from "../backend";
+import { Button, LabeledList, Box, AnimatedNumber, Section, Dropdown, Input, Table, Modal } from "../components";
+import { Window } from "../layouts";
+import { LabeledListItem } from "../components/LabeledList";
+import { createSearch, toTitleCase } from 'common/string';
+
+export const CargoConsole = (props, context) => {
```

Comment:
Looks good but this needs to be broken into multiple components e.g `Status`, `Browse` and `Details`.

Reply:
Any technical reason for this or just for styling

Reply:
Styling. Long functions aren't particularly agreeable to look at no matter the language.